app/models/form526_submission.rb

```diff
@@ -90,7 +90,14 @@ def rrd_processor_failed_handler(_status, options)
       submission.rrd_process_selector.send_rrd_alert(message)
       return submission.start_rrd_job(backup_processor_class)
     end
-
+    submission.start_evss_submission_job
+  rescue => e
+    message = <<~MESSAGE
+      RRD was skipped for submission #{submission.id} due to an error.<br/>
+      Sidekiq Job options: #{options}<br/>
+      Exception: #{e}<br/>
+    MESSAGE
+    submission.rrd_process_selector.send_rrd_alert(message)
     submission.start_evss_submission_job
```

Comment:
I could use `ensure`:
```ruby
ensure
  submission.start_evss_submission_job
```
but I didn't like that it may be misinterpreted as always run `submission.start_evss_submission_job`, which is not the case when the `return submission.start_rrd_job(backup_processor_class)` line is executed.